Formula/w/webdav.rb

```diff
@@ -0,0 +1,35 @@
+class Webdav < Formula
+  desc "Simple and standalone WebDAV server"
+  homepage "https://github.com/hacdias/webdav"
+  url "https://github.com/hacdias/webdav/archive/refs/tags/v5.8.0.tar.gz"
+  sha256 "fc9e5272c821f5a4ca98321195469a87a9ba8a8c696c0f4d3008407ee8150039"
+  license "MIT"
+  head "https://github.com/hacdias/webdav.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-s -w -X github.com/hacdias/webdav/v5/cmd.version=#{version}-brew"
```

Comment:
if people do `which webdav`, it would tell where it comes from, I dont think we need special version for brew?